src/components/layouts/Layout.js

```diff
@@ -64,6 +64,14 @@ const Layout = ({ children, initialContext, hasSideBar, location }) => {
     `}
       render={data => (
         <>
+          <div className="sprk-o-Box docs-c-SkipNav">
+            <a
```

Comment:
Why not use the sprk link component here instead of an a tag with the spark link classname added to it?